adminguide.rst

```diff
@@ -271,14 +271,20 @@ Example file installed path: ``/etc/flux/system/conf.d/system.toml``
  tcp_user_timeout = "2m"
 
  # Point to resource definition generated with flux-R(1).
- # Uncomment to exclude nodes (e.g. mgmt, login), from eligibility to run jobs.
+ # Uncomment 'norestrict' if flux broker is constrained to system cores by
+ #  systemd or other site policy.  This allows jobs to run on assigned cores.
+ # Uncomment 'exclude' to avoid scheduling jobs on certain nodes (e.g. login,
+ #  management, or service nodes).
```

Comment:
super tiny nit: are lines 275 and 277 supposed to have an extra space at the start of the line?

Reply:
As intended per RFC 7: 
> If @grondo authors a block comment, the first line of each paragraph SHALL be left justified, and the following lines SHALL be indented by one space.  If anyone else amends a @grondo block comment, it SHALL follow the same convention. 

Oh wait I think we forgot to add that.  :wink:

I'll just make it consistent with the rest of the file, thanks.

Reply:
Oh! Good to know, thanks for providing the reference too - TIL!

Reply:
Haha, funny. I just do that since it makes it a little easier to read comments. Sorry about violating the consistency of the file :-)